collections/_article/dev-snapshot-godot-4-2-beta-3.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Dev snapshot: Godot 4.2 beta 3"
+excerpt: "We continue iterating quickly on beta snapshots for Godot 4.2, to ensure that we can solve regressions before the stable release."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-4-2-beta-3.webp
+image_caption_title: The Last Game
+image_caption_description: A game by Frédéric Julian
+date: 2023-10-24 15:00:00
+---
+
+We entered the beta phase for Godot 4.2 a couple of weeks ago, with [4.2 beta 1](/article/dev-snapshot-godot-4-2-beta-1/). If you missed that release, have a look at the [release notes](/article/dev-snapshot-godot-4-2-beta-1/) for an overview of the key changes in Godot 4.2.
+
+Since the start of the beta phase, we fixed a number of bugs reported by the community, so we're publishing further beta snapshots to validate those fixes, and iterate closer to the release candidate stage.
+
+This third beta notably fixes a porting issue for 3D Godot 4.1 projects following our changes to the [mesh vertices and attributes compression format](https://godotengine.org/article/dev-snapshot-godot-4-2-beta-1/#rendering-particles-and-shaders). This beta provides a helpful dialog when you first open a pre-existing project, that lets you choose to upgrade and re-save all the meshes in your projects to make them compatible with Godot 4.2.
```

Comment:
~Hmm, what makes it a porting issue?~ Okay, I got it now, but at first I read "porting" in the platform support sense. Maybe there is a way to phrase it better.

Also two sentences in the row starting with "This beta".

```suggestion
Notably, this third beta improves support for 3D projects made with Godot 4.1 when migrating them to Godot 4.2. Following our changes to the [mesh vertices and attributes compression format](https://godotengine.org/article/dev-snapshot-godot-4-2-beta-1/#rendering-particles-and-shaders), we are now introducing a helpful dialog when you first open a pre-existing project. It lets you choose to upgrade and re-save all the meshes in your projects to make them compatible with Godot 4.2.
```